invenio_records_lom/config.py

```diff
@@ -26,6 +26,8 @@
     "json": {
         "name": _("JSON"),
         "serializer": "flask_resources.serializers:JSONSerializer",
+        "content-type": "application/json",
```

Comment:
To which task this change belong? (the same applies to the line below)

Reply:
This is part of the export-feature update (see further down comment).
Removed.